com.microsoft.java.lsif.core/src/com/microsoft/java/lsif/core/internal/LsifUtils.java

```diff
@@ -0,0 +1,33 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+package com.microsoft.java.lsif.core.internal;
+
+import java.util.Base64;
+
+import org.eclipse.core.runtime.Platform;
+
+public class LsifUtils {
+
+	private LsifUtils() {
+	}
+
+	/**
+	 * Normalize the URI to the same format as the client.
+	 */
+	public final static String normalizeUri(String uri) {
```

Comment:
cannot easily understand what it's doing from `normalizeUri`:

0-7: `file:///`
8: Upper Letter?   -> the purpose is to change this byte to lower?
9: `:`

Reply:
Basically it's dealing with the case of the driver on Windows. e.g.: Change `C:\\` to `c:\\`